agent/agentexec/cli_linux.go

```diff
@@ -56,28 +65,42 @@ func CLI() error {
 		}
 	}
 
-	if *oom == unset {
-		// If an explicit oom score isn't set, we use the default.
-		*oom, err = defaultOOMScore()
-		if err != nil {
-			return xerrors.Errorf("get default oom score: %w", err)
-		}
+	// We drop effective caps prior to setting dumpable so that we limit the
+	// impact of someone attempting to hijack the process (i.e. with a debugger)
+	// to take advantage of the capabilities of the agent process.
```

Comment:
You should mention that we encourage people to add some capabilities to the agent binary to improve network performance, and that having any capabilities sets dumpable to 0.